bulk_grades/api.py

```diff
@@ -331,6 +333,116 @@ def get_rows_to_export(self):
             yield row
 
 
+class InterventionCSVProcessor(CSVProcessor):
+    """
+    CSV Processor for intervention report grades.
+    """
+
+    required_columns = ['user_id', 'course_id']
+
+    def __init__(self, **kwargs):
+        """
+        Create InterventionCSVProcessor.
+        """
+        self.columns = ['user_id', 'username', 'email', 'student_key', 'full_name', 'course_id', 'track', 'cohort',
+                        'number of videos overall', 'number of videos last week', 'number of problems overall',
+                        'number of problems last week',
+                        'number of correct problems overall', 'number of correct problems last week',
+                        'number of problem attempts overall', 'number of problem attempts last week',
+                        'number of forum posts overall', 'number of forum posts last week',
+                        'date last active']
+        self.course_id = None
+        self.track = self.cohort = self._user = None
```

Comment:
Do we use `_user` for this processor? (Pretty sure it's only actually used in the other processor for generating history rows, so I suspect we don't.)